packages/mdl-icon-toggle/README.md

```diff
@@ -0,0 +1,199 @@
+# MDL Icon Toggle
+
+MDL Icon Toggle provides a material design icon toggle button. It is fully accessible, and is
```

Comment:
We also need to remember to note ligatures are not required to be used. Codepoints can also be used just fine (and required to avoid some issues like Google Translate breaking a page) in which case we should apply the `aria-label` attribute to define the help text.